README.md

```diff
@@ -219,11 +219,35 @@ the following contents:
 URL_ROOT = 'http://localhost:8000/a/{{DOMAIN}}'
 ```
 
+##### Running Formplayer in Docker
+
+Please refer to FormPlayer's install instructions under "[Running in Docker](https://github.com/dimagi/formplayer#running-in-docker)".
+
 If you are on Mac, don't bother trying to run this in Docker. There seems to be some kind of bug.
-Check out the install instructions under "[Building and Running]
-(https://github.com/dimagi/formplayer#building-and-running)".
+Instead, try running formplayer from a .jar file
+
+##### Running formplayer.jar
+
+Prerequisite: install Java (left as an exercise for the reader)
+
+To get set up, download the settings file and `formplayer.jar`. You may run this
+in the commcare-hq repo root.
+
+```bash
+$ curl https://raw.githubusercontent.com/dimagi/formplayer/master/config/application.example.properties -o formplayer.properties
+$ curl https://jenkins.dimagi.com/job/formplayer/lastSuccessfulBuild/artifact/build/libs/formplayer.jar -o formplayer.jar
```

Comment:
Maybe add a section for updating formplayer, or add a command to `hammer` to update it?

Reply:
Just added a note about updating.  Not sure it makes sense to add anything to the shared hammer, since not everyone runs formplayer this way (or needs to update it regularly)